src/lib.rs

```diff
@@ -214,6 +218,42 @@ impl Encoder {
 		Ok(value)
 	}
 
+	/// Configures the encoder's use of inband forward error correction (FEC).
+	pub fn set_inband_fec(&mut self, value: bool) -> Result<()> {
+		let value = match value {
+			false => 0,
+			true => 1
+		};
+		let result = unsafe { ffi::opus_encoder_ctl(self.ptr, OPUS_SET_INBAND_FEC, value) };
+		check("opus_encoder_ctl(OPUS_SET_INBAND_FEC)", result)
+	}
+
+	/// Gets encoder's configured use of inband forward error correction.
+	pub fn get_inband_fec(&mut self) -> Result<bool> {
+		let mut value: i32 = 0;
+		let result = unsafe { ffi::opus_encoder_ctl(self.ptr, OPUS_GET_INBAND_FEC, &mut value) };
+		try!(check("opus_encoder_ctl(OPUS_GET_INBAND_FEC)", result));
+		let value = match value {
+			0 => false,
+			_ => true
+		};
+		Ok(value)
+	}
+
+	/// Sets the encoder's expected packet loss percentage.
+	pub fn set_packet_loss_perc(&mut self, value: i32) -> Result<()> {
+		let result = unsafe { ffi::opus_encoder_ctl(self.ptr, OPUS_SET_PACKET_LOSS_PERC, value) };
+		check("opus_encoder_ctl(OPUS_SET_PACKET_LOSS_PERC)", result)
```

Comment:
Does opus return an error here if you pass it an out-of-range value? I would expect it to but it'd be nice if you could confirm.